app/assets/stylesheets/components/front/c-header.scss

```diff
@@ -420,7 +420,7 @@
               right: 0;
               top: 90%;
               margin: 5px 0 0;
-              width: calc(100% + 30px);
+              width: calc(100% + 40px);
```

Comment:
@clementprdhomme can you please confirm that this is gonna change the layout for the other templates as well?

Reply:
Yes.

@sureshkmit Could you use this instead?
```scss
ul {
  width: calc(100% + 30px);

  @if $theme == 3 {
    width: calc(100% + 40px);
  }
}
```

The variable `$theme` is defined in each  `_settings-theme-XXX.scss` file. It's assigned to the value `3` in the file `_settings-theme-ind.scss` so this code would only apply to the India template.